client/src/components/Author.js

```diff
@@ -67,12 +83,10 @@ class Author extends React.Component {
             <span>{name}</span>
           }
 
-          {!nodata && email && 
-              <span><br/><strong><a href={`mailto:${email}`} target='_blank' rel='noopener noreferrer'>{email}</a></strong></span>
+          {!nodata && emailElements
           }
 
-          {!nodata && website && 
-            <span><br/><strong><a href={website} target='_blank' rel='noopener noreferrer'>{website}</a></strong></span>
+          {!nodata && websiteElements
```

Comment:
This lists all emails first, then websites:

```
ethernaut@zeppelin.solutions
info@forta.org
https://openzeppelin.com
https://forta.org
```

We should list each email along with its website.

```
ethernaut@zeppelin.solutions
https://openzeppelin.com

info@forta.org
https://forta.org
```